devops/docker/smqtk_services/Dockerfile

```diff
@@ -54,23 +54,25 @@ RUN cd /caffe/build \
     -DUSE_CUDNN:BOOL=OFF \
     -DCMAKE_INSTALL_PREFIX:PATH=/caffe/install \
     /caffe/source \
- && make install -j12 \
+ && make install -j2 \
  && cd \
  && rm -r /caffe/source /caffe/build
 ENV PATH="/caffe/install/bin:${PATH}" \
     PYTHONPATH="/caffe/install/python:${PYTHONPATH}"
 
 # SMQTK
-ENV SMQTK_VERSION "master"
+ENV SMQTK_VERSION "v0.6.2"
```

Comment:
this is not needed, but realize if this is any version other than master, and especially one that is a tag that has a 'v' in it, when you download the archive from Github, it will untar into a folder without the 'v'. So you need the variable below to handle that.

Reply:
Thanks GitHub...